pkg/volume/iscsi/iscsi_util.go

```diff
@@ -448,3 +472,57 @@ func removeDuplicate(s []string) []string {
 	s = s[:len(m)]
 	return s
 }
+
+func parseIscsiadmShow(output string) (map[string]string, error) {
+	params := make(map[string]string)
+	slice := strings.Split(output, "\n")
+	for _, line := range slice {
+		if !strings.HasPrefix(line, "iface.") || strings.Contains(line, "<empty>") {
+			continue
+		}
+		iface := strings.Fields(line)
+		if len(iface) != 3 || iface[1] != "=" {
+			return nil, fmt.Errorf("Error: invalid iface setting: %v", iface)
+		}
+		// iscsi_ifacename can't be updated, skip
```

Comment:
more clarify, something like: `iscsi_ifacename` is immutable once the iface is created